packages/components/src/Input/index.tsx

```diff
@@ -163,6 +162,11 @@ function Input({
     onSubmit?.({ name, value: "" });
   };
 
+  const handlePassword = () => {
+    setsShowPassword(!showPassword);
+    document.getElementById(`input_${name}`)?.focus();
```

Comment:
SUGGESTION:
React suggests accessing the dom elements through their references, [here](https://reactjs.org/docs/hooks-reference.html#useref) is a guide on how to implement it

Reply:
it would look something like this:

```suggestion
    inputRef.current?.select();
```